src/modules/counter.js

```diff
@@ -0,0 +1,3 @@
+const counter = (item) => (item ? item.length : 0);
```

Comment:
- [ ] Please implement the comment counter in the way discussed earlier. 😎️

Reply:
My counter was called in pop-up.js and my data counter was not gotten from the API, It was gotten from the nodelist of ul, which is what was required.